src/interfaces/blobs.js

```diff
@@ -366,17 +366,22 @@ export default function ({url}) {
 				case transformationFailed:
 					return {
 						state: BLOB_STATE.TRANSFORMATION_FAILED,
+						modificationTime: moment(),
 						$set: {
 							'processingInfo.transformationError': payload.error
 						}
 					};
 				case transformationDone:
 					if ('numberOfRecords' in payload) {
+						if (payload.failedRecords) {
+							payload.failedRecords.push({failedrecordsTime: moment()});
```

Comment:
Don't work like this. `failedRecords` is an array of record-specific errors. Generate timestamp once and insert to each error so that the objects look like this: `{timestamp: '<timestamp>', error: '<the actual error'}`

Might require changes in commons...The transformers can decide the format of errors but the values in `failedRecords` are objects and the property `error` contains the error sent by the transformer.